Makefile

```diff
@@ -31,11 +31,21 @@ build-examples:
 
 .PHONY: build
 build:
+	go build $(BUILD_TAGS) $(filter-out ./core/auth/internal/gssapi,$(PKGS))
+
+.PHONY: build-no-tags
+build-no-tags:
 	go build $(filter-out ./core/auth/internal/gssapi,$(PKGS))
 
-.PHONY: build-cse
-build-cse:
-	go build -tags cse $(filter-out ./core/auth/internal/gssapi,$(PKGS))
+.PHONY: build-tests
```

Comment:
The structure here is kind of annoying. `go test -c` can only take a single package instead of a list, so we need to iterate over them and fail-fast if any of the packages fail to compile.